tests/lcs/core/engine/put.livecodescript

```diff
@@ -16,6 +16,38 @@ for more details.
 You should have received a copy of the GNU General Public License
 along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
 
+
+on _TestPutBeforeInvalidContainer
+    put "foo" before card 1 of this stack
+end _TestPutBeforeInvalidContainer
+
+on _TestPutIntoInvalidContainer
+    put "foo" into card 1 of this stack
+end _TestPutIntoInvalidContainer
+
+on _TestPutAfterInvalidContainer
+    put "foo" after card 1 of this stack
+end _TestPutAfterInvalidContainer
+
+on TestPutBeforeIntoAfterInvalidContainer
+	TestAssertThrow "put before invalid container" , "_TestPutBeforeInvalidContainer", the long id of me, \
+			"EE_CHUNK_SETNOTACONTAINER"
+	TestAssertThrow "put into invalid container" , "_TestPutIntoInvalidContainer", the long id of me, \
+			"EE_CHUNK_SETNOTACONTAINER"
+	TestAssertThrow "put after invalid container" , "_TestPutAfterInvalidContainer", the long id of me, \
+			"EE_CHUNK_SETNOTACONTAINER"
+end TestPutBeforeIntoAfterInvalidContainer
+
+on TestPutBeforeIntoAfterValidContainer
+    create button "myNewButton"
```

Comment:
We should do identical tests on for image, audioclip and videoclip - although audioclip will cause an error to be thrown at the moment due to https://quality.livecode.com/show_bug.cgi?id=7515.